vnext/Microsoft.ReactNative/DocString.h

```diff
@@ -5,7 +5,9 @@
 
 #ifdef USE_DOCSTRING
 #define DOC_STRING(x) [doc_string(x)]
+#define DOC_DEFAULT(x) [doc_default(x)]
```

Comment:
Nit: If marcros support this should we have a params? For multi-line? This would make the code easier to read and allow for sperate indentation of the lines and the comments in the line.
Now a doc comment fully folds to the left visually interrupting the nesting of the code....

Reply:
@dannyvv Sorry I didn't understand this comment, are you saying the macro should not take a param?